src/components/Accordion/Accordion.style.ts

```diff
@@ -0,0 +1,120 @@
+import { rem } from 'polished';
+import styled, { css } from 'styled-components';
+
+import { ChevronDown, CircleWarning } from '../../icons';
+import { Paragraph } from '../../typography';
+
+import { grayscale } from '../../color';
+import { core } from '../../tokens';
+
+export const AccordionStyled = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${rem(8)};
+`;
+
+export const Wrapper = styled.div<{
+  format: 'basic' | 'secondary';
+  active: boolean;
+  disabled: boolean;
+}>`
+  ${({ disabled }) =>
+    disabled &&
+    css`
+      pointer-events: none;
+      opacity: 0.4;
+    `}
+  ${({ active, theme, format }) =>
+    active &&
+    css`
+      background-color: ${format === 'basic'
+        ? theme.name === 'dark'
+          ? grayscale(800)
+          : grayscale(50)
+        : 'none'};
+    `}
+  border-radius: ${rem(10)};
+  color: ${core.color.text.primary};
+  ${({ active, theme, format }) =>
+    active &&
+    css`
+      border: ${format === 'secondary'
+        ? theme.name === 'dark'
+          ? `${rem(1)} solid ${grayscale(800)}`
+          : `${rem(1)} solid ${grayscale(50)}`
+        : 'none'};
+    `}
+`;
+
+export const TriggerContainer = styled.button<{
+  format: 'basic' | 'secondary';
+  active: boolean;
+}>`
+  display: flex;
+  justify-content: space-between;
+  cursor: pointer;
+  padding: ${rem(12)} ${rem(15)};
+  border-radius: ${rem(10)};
+  width: 100%;
+  &:hover {
+    background-color: ${({ theme, format }) =>
+      format === 'basic'
+        ? theme.name === 'dark'
+          ? grayscale(800)
+          : grayscale(50)
+        : 'none'};
+    outline: ${({ active, theme, format }) =>
+      !active && format === 'secondary'
+        ? theme.name === 'dark'
+          ? `${rem(1)} solid ${grayscale(800)}`
+          : `${rem(1)} solid ${grayscale(50)}`
+        : 'none'};
+  }
+`;
+
+export const Header = styled.div`
+  display: flex;
+  margin-right: ${rem(10)};
+`;
+
+export const TitleContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+`;
+
+export const Title = styled(Paragraph)`
+  font-size: ${rem(16)};
```

Comment:
Should this be a [Header](https://vimeo.github.io/iris/sb/DSYS-8-toggle/?path=/docs/typography-header--docs) component?